### ISSUE TITLE
Unable to create Khepri projection when running Mix release

### ISSUE THREAD

**Issue by xh4** [rabbitmq/khepri#212]
### What does not work?

The following code works when I use `mix run`:
```elixir
:khepri.start()

projection_name = :foo
path_pattern = "/foo/*"
projection_fun = fn (path, value) -> {path, value} end
options = %{type: :set, read_concurrency: true}
projection = :khepri_projection.new(projection_name, projection_fun, options)
:khepri.register_projection(path_pattern, projection)
```

But when running a release, the following exception was raised:

```
16:04:09.346 [notice] Application hello exited: exited in: Hello.Application.start(:normal, [])
    ** (EXIT) an exception was raised:
        ** (ErlangError) Erlang error: {:horus_ex, :module_not_found, %{module: :erlang}}
            (horus 0.2.3) src/horus.erl:1984: :horus.get_object_code_from_code_server/1
            (horus 0.2.3) src/horus.erl:1927: :horus.disassemble_module1/2
            (horus 0.2.3) src/horus.erl:1879: :horus.disassemble_module/2
            (horus 0.2.3) src/horus.erl:1778: :horus.lookup_function/4
            (horus 0.2.3) src/horus.erl:1155: :horus.pass1_process_function/4
            (stdlib 5.0.2) maps.erl:416: :maps.fold_1/4
            (horus 0.2.3) src/horus.erl:745: :horus.extract_module_info_functions/1
            (horus 0.2.3) src/horus.erl:497: :horus.to_standalone_fun3/2
```

### Expected behavior

Khepri projection should work in a Mix release.

### How to reproduce

Erlang/OTP 26 [erts-14.0.2]
Elixir 1.15.4

**Comment by xh4** [comment]
Yes, this fixes the problem. Thanks.

**Comment by dumbbell** [comment]
Horus 0.2.4 and the just released Khepri 0.8.0 include a fix for this problem. You should no longer need to work around the issue from your `mix.exs`.
